### ISSUE TITLE
R1: Demo branch for development

### ISSUE THREAD

**Issue by ConorTighe1995** [ospin-web-dev/REACTor#738]
blocked by: ospin-web-dev/hambda#432

### Feature Description / Problem Solved by Feature
As a developer, I want to keep the version of the Ospin web application for the demo separate. This is to avoid entangling the complexities of the features associated with this project with the current web app in production. 

(this below should maybe be a separate ticket to be done once MVP is complete? I am unsure. We might not want to ever merge the demo branch -ds)
When the branch reaches a stable state in the future a pre-release review will be held. This is a meeting involving a code walk through with the team discussing the files and lines of code that will be adding to the production application to minimize technical debt and error when releasing the demo mode.

---

### Deliverables
As a developer...
 - [ ] I can git pull and branch etc. from the demo branch
 - [ ] As a developer, merged PRs into the demo branch trigger the CI/CD pipeline and deploy to `demo.ospin-app.com`
 - [ ] As a developer I can see a header and short description at the top of the README of this branch detailing that this is the demo branch and what its for
